libevmasm/Assembly.h

```diff
@@ -152,6 +152,12 @@ class Assembly
 		bool _includeSourceList = true
 	) const;
 
+	/// Loads the JSON representation of assembly.
+	/// @param _json JSON object containing assembly
+	/// @param _loadSources true, if source list should be included, false otherwise.
+	/// @returns true on success, false otherwise
+	bool loadFromAssemblyJSON(Json::Value const& _json, bool _loadSources = true);
```

Comment:
Shouldn't this be a static factory function instead?

Reply:
Yeah that might be better.